apps/oidc-admin-angular/src/styles.scss

```diff
@@ -1135,3 +1135,15 @@ th {
 tr:nth-child(even) {
   background-color: #f7f7f7;
 }
+
+.form-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 2rem;
+
+  tamu-gisc-select {
+      margin-left: 0.5rem;
+      margin-right: 0.5rem;
+  }
+}
```

Comment:
Run formatter. Inconsistent line endings.

Reply:
So that changed 27 files across `gisday` and `oidc` apps and libs. I have the formatter setup to run on save, how could so many files need to be formatted again?

Reply:
Unsure, but it happens.